### PR TITLE
Add Q&A about moving from cloud to self-hosted/etc

### PR DESCRIPTION
**Pull request by ulysseskan**
Fulfill https://github.com/gravitational/cloud/issues/343

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: docs/pages/cloud/faq.mdx [hunk 1]
```diff
@@ -21,6 +21,10 @@ this number reflects risks in our architecture that we will improve over time.
 
 [Reach out to sales](https://goteleport.com/signup/enterprise) to discuss pricing.
 
+## If I start with Teleport Cloud, can I move to self-hosted or community, or do I need to start again?
```
Comment (ptgott) [inline docs/pages/cloud/faq.mdx:24]:
```suggestion
## If I start with Teleport Cloud, can I move to Teleport Enterprise or Teleport Open Source, or do I need to start again?
```

File: docs/pages/cloud/faq.mdx [hunk 2]
```diff
@@ -21,6 +21,10 @@ this number reflects risks in our architecture that we will improve over time.
...
+
+If you also plan to use S3 and DynamoDB, we can provide data for you to import. But you should reach out to us first.
```
Comment (ptgott) [inline docs/pages/cloud/faq.mdx:26]:
```suggestion
If you plan to use S3 and DynamoDB as storage backends, we can provide data for you to import. But you should reach out to us first.
```

Comment (ptgott) [reply docs/pages/cloud/faq.mdx:26]: To make this a bit more explicit, I think we should also add a sentence indicating what happens if you plan to use another storage backend.

**Review by ptgott — APPROVED** [review]

File: docs/pages/cloud/faq.mdx (hunk 2 above)
Comment (ulysseskan) [reply docs/pages/cloud/faq.mdx:26]: Maybe "If you use a different backend, you will need to start over."?  I don't know if we have any migration path from the Cloud's S3/DynamoDB to other backends.

Comment (ptgott) [reply docs/pages/cloud/faq.mdx:26]: Works for me!

**Pull request merged**